### DISCUSSION TITLE
Javascript learning

### DISCUSSION THREAD

**Discussion by Rickyngechu** [codewars/codewars.com#2951] [General]
Any beginner into javascript,aspiring to be a javascript developer. 
Well I have a discord channel where we developers can get job opportunities,learn and even brainstorm some useful ideas.If intrested I will share the link
You can now join my discord channel through this link .  https://discord.gg/JpjQ8t8H

**Comment by Ktopper** [comment]
I am. Been really hitting hard for two months but have been learning for 6 months. I've been learning Html and Css too so I can do some front end development in the future.

**Reply by strawberiix** [reply]
hai

**Comment by jetskeeter1** [comment]
I'm also learning javascript. I struggle to get a balance with my college and self development, but at the same time I am determined to find a way to get things done for school and still have enough time to learn Javascript.

**Reply by Aghaei-Dev** [reply]
im doing the same but im learning react with all of them

**Comment by WokeCat** [comment]
Im also a beginner trying to learn JavaScript. At the moment im creating a discord bot as a side project to practice coding, Once I have a better understanding of js i hope to learn R and python and go into data science a bit more but that is still way off in the future.

**Reply by Rickyngechu** [reply]
Hello,same case applies to me.There is a discord group where we can learn,create challenges and even projects or even job opportunities.I will share the link here.

**Comment by 7usien** [comment]
try to read here :

https://javascript.info/
https://www.freecodecamp.org/news/search/?query=javascript

**Comment by DreamchaserFF** [comment]
I've been learning Js and SQL for... a while now. But it's been difficult to juggle studying code with 10 to 12 hours shifts on a job I hate. But we'll keep on keeping on.

**Comment by Desmond09-spec** [comment]
I'm also a beginner to JavaScript and I'm learning html and css and also some frameworks like electron and react and it's been tough as  I struggle to balance it with my studies but I won't give up.

**Comment by kes-crypto** [comment]
> Any beginner into javascript,aspiring to be a javascript developer.

Did you get a job as a javascript developer? Any leads to get an entry job would be much appreciated.

**Comment by Blind4Basics** [comment]
@pravinmohite no self promotion, thx.

**Comment by Deepikaa123** [comment]
> Any beginner into javascript,aspiring to be a javascript developer. Well I have a discord channel where we developers can get job opportunities,learn and even brainstorm some useful ideas.If intrested I will share the link

Yes please

**Reply by Rickyngechu** [reply]
https://discord.gg/JpjQ8t8H
